12/umbraco-cms/reference/api-versioning-and-openapi.md

```diff
@@ -0,0 +1,346 @@
+---
+description: How to use API versioning and OpenAPI (Swagger) for your own APIs.
+---
+
+# API versioning and OpenAPI (Swagger)
+
+Umbraco 12 ships with Swagger to document the Content Delivery API. Swagger and the Swagger UI is available at `{yourdomain}/umbraco/swagger`. For security reasons, both are disabled in non-production environments.
```

Comment:
Removed "12" as this will quickly become redundant.

Also, in case someone doesn't know what Swagger is, should we consider linking off to something that could explain this? Or perhaps add a few more words to it here?

```suggestion
Umbraco ships with Swagger to document the Content Delivery API. Swagger and the Swagger UI is available at `{yourdomain}/umbraco/swagger`. For security reasons, both are disabled in non-production environments.
```

Reply:
@sofietoft this article is only relevant for Umbraco 12 and above - previous versions do not ship with Swagger. But I suppose since this article will only be part of the 12+ docs, it doesn't matter much if we remove the version notation here...? If it ends up confusing people on older versions, we can always amend it to "Umbraco 12+". 

Also - people reading this article will know what Swagger is 😄